controlplane/pkg/nsm/heal_constants.go

```diff
@@ -0,0 +1,8 @@
+package nsm
+
+import "time"
+
+const (
+	HealTimeout          = time.Minute * 10
```

Comment:
What are these numbers? Please add some comments here and explain. 10 minutes look too much to me.

Reply:
I'm also curious about why 10 minutes...

Reply:
Let's have it 1 minute for now, in case some long pod-restarts.